xml/klp.xml

```diff
@@ -295,10 +295,10 @@ Activate with: SUSEConnect -p sle-module-live-patching/&productnumber-regurl;/x8
      When the <package>kernel-default</package> package is installed, the update
      manager prompts you to reboot the system. To prevent this message from
      appearing, you can filter out kernel updates from the patching operation.
-     This can be done by adding package locks with Zypper. &susemgr; also makes
+     This can be done by adding package locks with Zypper. &smlm; also makes
      it possible to filter channel contents (see
-     <link xlink:href="https://documentation.suse.com/suma/4.3/en/suse-manager/administration/live-patching.html">Live
```

Comment:
Adjusted pointer to latest release version of the docs. Might need to be updated after SMLM 5.1 is released.

Reply:
Not that I would know of...